incubator/hnc/plugin/hnc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: hnc
+spec:
+  version: "v0.0.1"
+  platforms:
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: https://github.com/kubernetes-sigs/multi-tenancy/releases/v0.0.1.tar.gz
+      sha256: "5c7b88db640b12f0fb4794392e7b9574115a3814e9783966d56144449d459c49"
```

Comment:
Ditto, this should be some dummy value like `SHA256` and be replaced by the _actual_ value during the build process.